packages/node-client/lib/types.ts

```diff
@@ -6,6 +6,7 @@ export enum AuthModes {
     Basic = 'BASIC',
     ApiKey = 'API_KEY',
     AppStore = 'APP_STORE',
+    Custom = 'CUSTOM',
```

Comment:
should we pick a more specific name to better describe what this auth mode does?